app/src/main/java/net/bible/android/view/activity/page/screen/DocumentViewManager.kt

```diff
@@ -82,34 +79,25 @@ class DocumentViewManager @Inject constructor(
 
     @Synchronized
     fun buildView(forceUpdate: Boolean = false) {
-        if (myNoteViewBuilder.isMyNoteViewType) {
+        val view = buildWebViews(forceUpdate)
+        if(lastView != view) {
             removeView()
-            mainBibleActivity.resetSystemUi()
-            lastView = myNoteViewBuilder.addMyNoteView(parent)
-        } else {
-            val view = buildWebViews(forceUpdate)
-            if(lastView != view) {
-                removeView()
-                lastView = view
-                parent.addView(view,
-                    LinearLayout.LayoutParams(
-                        ViewGroup.LayoutParams.MATCH_PARENT,
-                        ViewGroup.LayoutParams.MATCH_PARENT)
-                )
-            }
-            ABEventBus.getDefault().post(WebViewsBuiltEvent())
+            lastView = view
+            parent.addView(view,
+                LinearLayout.LayoutParams(
+                    ViewGroup.LayoutParams.MATCH_PARENT,
+                    ViewGroup.LayoutParams.MATCH_PARENT)
+            )
         }
+        ABEventBus.getDefault().post(WebViewsBuiltEvent())
     }
 
     val documentView: DocumentView?
         get() = getDocumentView(windowControl.activeWindow)
 
-    private fun getDocumentView(window: Window): DocumentView? {
-        return if (myNoteViewBuilder.isMyNoteViewType) {
-            myNoteViewBuilder.view
-        } else { // a specific screen is specified to prevent content going to wrong screen if active screen is changed fast
-            splitBibleArea?.bibleViewFactory?.getOrCreateBibleView(window)
-        }
+    fun getDocumentView(window: Window): DocumentView? {
+        // a specific screen is specified to prevent content going to wrong screen if active screen is changed fast
+        return splitBibleArea?.bibleViewFactory?.getOrCreateBibleView(window)
```

Comment:
In the future this function could become more complex again, so I'm not in favour of this change.

Reply:
If a function is one-liner that returns a value, just press alt+enter on top of "return" and it offers to change it to "expression body" form. And as easily  it can be changed back to "body block" form if one day it needs to be made more complex. Another thing here was that I believe the comment is outdated or otherwise unclear, so better remove it.